app/controllers/verify/review_controller.rb

```diff
@@ -25,31 +26,15 @@ def new
     end
 
     def create
-      resolution = start_idv_session
-      track_idv_event(resolution)
-      process_resolution(resolution)
+      init_questions_and_profile
+      redirect_on_success
+      analytics.track_event(Analytics::IDV_REVIEW_COMPLETE)
     end
 
     private
 
-    def track_idv_event(resolution)
-      result = {
-        success: resolution.success,
-        idv_attempts_exceeded: idv_attempter.exceeded?
-      }
-
-      analytics.track_event(Analytics::IDV_INITIAL, result)
-    end
-
-    def process_resolution(resolution)
-      if resolution.success
-        init_questions_and_profile(resolution)
-        redirect_on_success
-      elsif idv_attempter.exceeded?
-        redirect_to verify_fail_path
-      else
-        redirect_to verify_retry_path
-      end
+    def idv_profile_complete?
+      idv_session.resolution.present? && idv_session.resolution.success?
```

Comment:
I tried installing the Code Climate browser plugin like @monfresh suggested, and it's 💯 , it's highlighting: "Verify::ReviewController#idv_profile_complete? calls 'idv_session.resolution' 2 times"

Maybe this could be:

```ruby
idv_session.resolution.try(:success?)
```

or the new-fangled &. syntax?